pkg/memio/ports_linux.go

```diff
@@ -29,3 +42,55 @@ func Out(addr uint16, data UintN) error {
 	}
 	return pathWrite(portPath, int64(addr), data)
 }
+
+func init() {
+	ioplError = syscall.Iopl(3)
```

Comment:
Wouldn't this mean every command in the busybox has Iopl(3)?

Also, if someone just imported the memio package without intending to use this, it would be surprising to them to see iopl(3) being called at the program start.

Reply:
Fixed. see what you think.